src/App.tsx

```diff
@@ -1,39 +1,73 @@
-import React from 'react';
+import { Component } from 'react';
 import './App.scss';
+import { Clock } from './components/clock';
+
+type State = {
+  clockName: string,
+  visibleClock: boolean,
```

Comment:
For boolean values it is better to give names starting with is/are/can etc. so it's clear that value has boolean type.
![image](https://user-images.githubusercontent.com/96409563/195268353-8fb4a78b-177b-4595-bdaf-e81103c6f2f9.png)